matminer/featurizers/composition.py

```diff
@@ -1391,6 +1409,35 @@ def __init__(self):
         # Load in a table of elemental properties
         self.elem_data = MagpieData()
 
+        element_list = [
```

Comment:
Would it make sense to move this list as a class attribute of MagpieData?

Reply:
Well this list is just a Yang-specific subset of elements in MagpieData. So it is not really specific to MagPie data, more specific to Yang. I mean we could add a .yang attr to MagpieData, but imo it is probably clearer being in the YangSS class itself.

Reply:
Sorry, should have read the code more. This seems like a better attribute to add to MixingEnthalpy? Maybe a "get_supported_elements" function? The MixingEnthalpy class is the one that seems to impose limits on which materials could be featurized.

Reply:
Sure, added. Thanks @WardLT !